tools/transformer/src/transformations/tagExperimental.js

```diff
@@ -0,0 +1,27 @@
+const stableOperationIds = require("../operations.stable.json").stableIds;
+
+/**
+ * Transforms the path description to mark paths as "experimental" where paths are not flagged as stable.
+ * Transformation prepends the tag "(experimental)" at the start of the path description
+ *
+ *
+ * @param {*} api OpenAPI JSON File
+ * @returns OpenAPI JSON File
+ */
+function applyAddExperimentalToDescriptions(api) {
+  Object.values(api.paths).forEach((path) => {
+    Object.values(path).forEach((requestMethod) => {
+      if (
+        requestMethod.operationId &&
+        !stableOperationIds.includes(requestMethod.operationId)
+      ) {
+        requestMethod.description = `(experimental) ${requestMethod.description}`;
```

Comment:
[Nit] optional suggestion

```suggestion
        requestMethod.description = `[experimental]${requestMethod.description}`;
```